src/components/MultiHopTrade/hooks/useAccountIds.tsx

```diff
@@ -12,10 +18,34 @@ export const useAccountIds = (): {
 } => {
   const dispatch = useAppDispatch()
 
-  // currently, a multi-hop trade will be automagically routed though the accountId corresponding to
-  // the accountNumber for the first hop.
```

Comment:
That was the intent, but wasn't working as intended - comment reworded moved on top of `buyAssetAccountId`